api.rb

```diff
@@ -113,7 +113,8 @@ class Api < Roda
   end
 
   def render_with_games
-    render(pin: request.params['pin'], games: Game.home_games(user, **request.params).map(&:to_h))
+    invite = request.params['invite'].to_i if request.params['invite']
+    render(pin: request.params['pin'], games: Game.home_games(user, **request.params).map(&:to_h), invite: invite)
```

Comment:
let's just add all params  so remove invite and pin and do render(**request.params)

Reply:
or instead, can we just create an actual invite route and not use params? 

so it should be /invite/1